lib/grit.rb

```diff
@@ -62,6 +62,15 @@ class << self
     def log(str)
       logger.debug { str }
     end
+
+    def reencode_string(str)
+      target_encoding = Encoding.default_internal || Encoding.default_external
```

Comment:
We set `default_internal` explicitly in the app (or Rails does, maybe), but in a default `irb` it's `nil` (but `default_external` is set), so I thought it best to prefer `internal` if it's set, but defer to `external` otherwise.